p2p/host/resource-manager/conn_limiter.go

```diff
@@ -1,13 +1,26 @@
 package rcmgr
 
 import (
+	"math"
 	"net/netip"
+	"slices"
 	"sync"
 )
 
-type ConnLimitPerCIDR struct {
-	// How many leading 1 bits in the mask
-	BitMask   int
+type ConnLimitPerSubnet struct {
+	// This defines how big the subnet is. For example, a /24 subnet has a
+	// BitMask of 24. All IPs that share the same 24 bit prefix are in the same
+	// subnet.  Are in the same subnet, and bound to the same limit.
+	BitMask int
```

Comment:
Is `PrefixLength` better? 
bitmask suggest's it is the network's bitmask that you can `AND` the ip address with.